src/main/java/com/thagorn/hex/hexserver/model/Level.java

```diff
@@ -27,9 +27,8 @@ public class Level {
     @NonNull
     String name;
     String description;
-    @ManyToOne(cascade = CascadeType.PERSIST)
-    private User owner;
+    // @ManyToOne(cascade = CascadeType.PERSIST)
+    // private Game game;

```

Comment:
It's generally considered poor form to check in commented out code, I think you can just delete this